projects/core/src/lib/colors/color-palette-editor/color-palette-editor.component.html

```diff
@@ -0,0 +1,13 @@
+<cdk-virtual-scroll-viewport appendOnly itemSize="80" class="tabs-viewport">
+    <div *cdkVirtualFor="let c of allColors; let i = index" class="grid-container">
+        <geoengine-color-attribute-input
+            class="grid-colorpicker"
+            [ngModel]="c"
+            (ngModelChange)="rebuildColorMap(i, $event)"
+        ></geoengine-color-attribute-input>
+        <button mat-icon-button (click)="removeColorTab(i)" class="grid-removebutton"><mat-icon>remove</mat-icon></button>
+        <div class="grid-errormsg" *ngIf="notFloat(i)">Value must be a number!</div>
+    </div>
+</cdk-virtual-scroll-viewport>
+<br />
+<button mat-raised-button color="accent" (click)="addColorTab()">Add color tab</button>
```

Comment:
For NDVI, when you click add, the table jumps to a position where the new entry isn't visible.

Reply:
Fixed it so the table will always jump to the (new) bottom entry when clicking add [720a718](https://github.com/geo-engine/geoengine-ui/pull/614/commits/720a718fd695adfa16c1884236e53a86b7901094)

Reply:
Works in Chromium, but not in FF. Should be related to the scrollbar issue.

Reply:
This got solved alongside the scroll bar issues in [7fbcc6e](https://github.com/geo-engine/geoengine-ui/pull/614/commits/7fbcc6edc42516577798dfccc44d9fedc99f57fd)